skyrim-platform/tools/plugin-example/LICENSE.txt

```diff
@@ -1,21 +0,0 @@
-MIT License 
```

Comment:
SP is distributed under GPLv3 so as far as this change is ok, we should somehow point that plugin-example isn't polluted by GPL restrictions. We need to somehow say that they are able to do whatever they want with this bunch of configs and these three lines of code.

Reply:
I added a bit about licensing under a `LICENSE` section at the bottom

> ### LICENSE
>
> Your plugin is your own software and can be licensed however you like!
> 
> Skyrim Platform is distributed under the GNU General Public License v3.0.